### PR TITLE
updating cypress and fixing tests for staging

### PR DESCRIPTION
**Pull request by amyreit**
Upgrading cypress level and updating tests to run successfully manually merging changes from draft to staging

### CODE REVIEW

**Review by aknguyen7 — COMMENTED** [review]
the ???.cy.js name confused me, is that for each of diff test name file?

**Comment by amyreit** [comment]
The new version of cypress changes the test files from *.js to *.cy.js and puts them in the e2e directory not the integration directory

File: src/test/cypressjs/README.md [hunk 1]
```diff
@@ -4,13 +4,17 @@
...
-- **Option 1 from step 2** using `cypress.env.json`
-  - `npx cypress open`
-- **Option 2 from step 2** using CLI variable instead of `cypress.env.json`
-  - `npx cypress open --env website_url=https://openliberty.io`
+NOTE - do not check in the cypress.config.js file with any changes for your local environment   
+
+## Step 3:  Launch the Cypress test runner or run headless
+
+- `npx cypress open` to run in the ui
+- `npx cypress run --spec "cypress/e2e/???.cy.js"` to run headless
```
Comment (aknguyen7) [inline src/test/cypressjs/README.md]: is this command npx cypress run --spec "cypress/e2e/???.cy.js gonna run all tests? with the ???.cy.js?

Comment (amyreit) [reply src/test/cypressjs/README.md]: ??? is where you substitute the actual test name you want to run - I can make that more clear

Comment (amyreit) [reply src/test/cypressjs/README.md]: done

**Review by aknguyen7 — APPROVED** [review]

**Pull request merged**